src/Workspaces/Core/Portable/Classification/SyntaxClassification/AbstractEmbeddedLanguageTokenClassifier.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using Microsoft.CodeAnalysis.EmbeddedLanguages.LanguageServices;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.Classification.Classifiers
+{
+    internal abstract class AbstractEmbeddedLanguageTokenClassifier : AbstractSyntaxClassifier
```

Comment:
>AbstractEmbeddedLanguageTokenClassifier [](start = 28, length = 39)

nit: should this be plural `AbstractEmbeddedLanguagesTokenClassifer`?

Reply:
yup